devicefilter/devicefilter.go

```diff
@@ -52,8 +53,8 @@ func (f *DeviceFilter) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	lanternDeviceId := req.Header.Get(deviceIdHeader)
 
 	if lanternDeviceId == "" {
-		f.log.Debugf("No %s header found, respond 404 not found to %s\n", deviceIdHeader, req.RemoteAddr)
-		w.WriteHeader(http.StatusNotFound)
+		f.log.Debugf("No %s header found from %s, mimicking apache\n", deviceIdHeader, req.RemoteAddr)
+		mimic.MimicApache(w, req)
```

Comment:
I read below in the comments that it actually mimics an unconfigured Apache. Should we convey that in the naming?